dashboard/src/app/stacks/stack-details/stack.html

```diff
@@ -47,7 +47,7 @@
                     editor-read-only="true"
                     editor-mode="text/x-yaml">
         </che-editor>
-          <a target="_blank" href="https://redhat-developer.github.io/devfile/">Docs: Devfile Structure</a>
+          <a ng-href="{{branding.docs.stack}}" target="_blank">Docs: Devfile Structure</a>
```

Comment:
Where this link is going?

Reply:
This link is going to "/docs/creating-starting-workspaces.html", then redirect to "/docs/che-6/creating-starting-workspaces.html" because Che 7 part doesn't exist in https://github.com/eclipse/che-docs yet.

We have a good description of the devfile here "https://redhat-developer.github.io/devfile/". This link was in the first implementation.

Our devfiles are the Che 7  future for stacks. That is why I added a link for the stack documentation.

Let's discuss what should we do:
- Live this link as is.
- Remove this link at al.
- Something else.

We have an issue for a docs part https://github.com/eclipse/che/issues/13976

WDYT?

Reply:
What? docs for Che 7 don't exist? they are.. 
https://github.com/eclipse/che-docs/tree/master/src/main/pages/che-7

But I would recommend you to give a look on the published one:
https://www.eclipse.org/che/docs/che-7/index.html

There is one page which should trigger your attention:
https://www.eclipse.org/che/docs/che-7/using-developer-environments-workspaces.html#making-a-workspace-portable-using-a-devfile_using-developer-environments-workspaces

Reply:
Sorry for this

Reply:
I was confusing with an old docks structure. 
Thanks for your help with the needed docs link.